packages/dd-trace/test/appsec/iast/code_injection.integration.spec.js

```diff
@@ -27,7 +27,7 @@ describe('IAST - code_injection - integration', () => {
 
   after(async function () {
     this.timeout(60000)
-    await sandbox.remove()
+    await sandbox?.remove()
```

Comment:
Drive-by fix: if the before would fail, sandbox is going to be undefined. This came up in a CI, so we just fixed that and increased the timeout a tad.